service/src/test/java/edu/softserve/zoo/service/TaskServiceTest.java

```diff
@@ -0,0 +1,36 @@
+package edu.softserve.zoo.service;
+
+import edu.softserve.zoo.exceptions.ApplicationException;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.test.context.ActiveProfiles;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.junit4.SpringJUnit4ClassRunner;
+import org.springframework.transaction.annotation.Transactional;
+
+/**
+ * @author Taras Zubrei
+ */
+@RunWith(SpringJUnit4ClassRunner.class)
+@ContextConfiguration("classpath*:spring/service-test-ctx.xml")
+@ActiveProfiles("test")
+@Transactional
+public class TaskServiceTest {
```

Comment:
I did it in such way because the only difference between service layer and persistence is checking of employee in database. So positive cases are covered in persistence tests